### ISSUE TITLE
p<c-p>

### ISSUE THREAD

**Issue by coachshea** [svermeulen/vim-easyclip#74]
I am a new user and I am so far loving it. My only confusion is that when I press `p<c-p>` I keep getting the message "Last action was not a paste, swap ignored" I do this immediately after pasting. The pasting works as expected, but always this message. Any help is greatly appreciated.

**Comment by svermeulen** [comment]
I can't seem to reproduce this problem.  Could you try it with a very minimal `.vimrc` to see if it still occurs?  It might be a conflict with some other vimscript you have

**Comment by svermeulen** [comment]
Whenever I hit `p<c-p>` it works fine and correctly cycles through the yank buffer.  You only get that error if you do a change operation after paste and before `<c-p>`.  Like, if you paste, then hit `o` to add a new line, then hit `<c-p>` you'll get that error

**Comment by coachshea** [comment]
Thank you for the quick response. Unfortunately that's not what's happening to me. I am pressing <c-p> immediately after p and I'm still getting the error. 

On January 11, 2016, at 1:03 PM, Steve Vermeulen <notifications@github.com> wrote:

Whenever I hit p<c-p> it works fine and correctly cycles through the yank buffer. You only get that error if you do a change operation after paste and before <c-p>. Like, if you paste, then hit o to add a new line, then hit <c-p> you'll get that error

**Comment by coachshea** [comment]
That worked so obviously it is something else in my setup. Any idea what I 
should be looking for?

**Comment by svermeulen** [comment]
I'm not sure.  I would try slowly adding back plugins / vimscript until you reproduce the issue again, to narrow it down.

**Comment by coachshea** [comment]
I want to thank you for all of your help. In the unlikely event that this 
happens to someone else, I figured I'd share my issues. I had installed a 
plugin that autosaved my files on TextChanged. Every paste of course fired the 
event and the save was the intervening event that prohibited this plugin from 
doing its job. I had been using this for a while, so it didn't immediately 
occur to me that this was happening, but that's what it was.

Thanks again for your help.
